CONTRIBUTING.md

```diff
@@ -4,6 +4,54 @@ After checking out the repo, run `bin/setup` to install dependencies. You can al
 
 To install this gem onto your local machine, run `bundle exec rake install`. To release a new version, update the version number in `version.rb`, and then run `bundle exec rake release`, which will create a git tag for the version, push git commits and tags, and push the `.gem` file to [rubygems.org](https://rubygems.org).
 
+## Running tests
+
+The gem test suite can be run in two modes. By default, it runs against the
+live sandbox environment, creating items on the fly and calling various API
+endpoints for them. For this to work you'll need real `client_id`, `secret`,
+and `public_key` from your Plaid dashboard. Create a file named `env.sh` under
+`tmp/`:
+
+```bash
+#!/bin/sh
+
+set -e
+
+export PLAID_RUBY_CLIENT_ID=the_real_client_id
+export PLAID_RUBY_SECRET=the_real_secret
+export PLAID_RUBY_PUBLIC_KEY=the_real_public_key
+
+exec "$@"
+```
+
+Make this file executable (`chmod +x tmp/env.sh`), and you'll be able to run tests with
+`tmp/env.sh rake test`.
```

Comment:
Hi @be9! May I suggest using the [Dotenv gem](https://github.com/bkeepers/dotenv) for this purpose, instead? Within the repo you could supply a `.env.sample` (and add `.env` to `.gitignore`).